docs/pages/abide.md

```diff
@@ -34,6 +34,18 @@ These input types create a text field: `text`, `date`, `datetime`, `datetime-loc
       <p class="help-text" id="exampleHelpTex">This input is ignored by Abide using `data-abide-ignore`</p>
     </div>
     <div class="small-12 columns">
+      <label>Disabled!
+        <input type="text" placeholder="You can't use me!" aria-describedby="exampleHelpTex" disabled>
```

Comment:
Also, don't you think there is already too much inputs in this example, and we should rather move ignored input to their own section ?